sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -673,9 +673,11 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
             }
 
           case p @ Project(exprs, _) if containsMultipleGenerators(exprs) =>
-            p.failAnalysis(
-              errorClass = "_LEGACY_ERROR_TEMP_2433",
-              messageParameters = Map("sqlExprs" -> exprs.map(_.sql).mkString(",")))
+            throw SparkException.internalError(
+              msg = "Only a single table generating function is allowed in a SELECT clause, " +
```

Comment:
is this really internal? does the analyzer split `select explode(...), explode(...) ...` into two Projects?

Reply:
Let me have a try.